fl4health/utils/losses.py

```diff
@@ -221,15 +221,16 @@ def aggregate_losses_dict(
             Dict[str, torch.Tensor]: A single dictionary with the aggregated losses according to the given loss
                 meter type
         """
-        # We don't know the keys of the dict (backward or additional losses) beforehand. We obtain them
-        # from the first entry because we know all of the losses will have the same keys
-        loss_keys = loss_list[0].keys()
-        num_losses = len(loss_list)
+        # We don't know the keys of the dict (backward or additional losses) beforehand. We don't obtain them
```

Comment:
The reason that it is working because I have decouple mkmmd implementation from moon and I just didn't include it in that PR to keep things simple. Maybe I will put up another PR for that.

Reply:
I think maybe I'm having trouble picturing where this comes up. I vaguely recall we might have talked about this, but I've since forgotten. Maybe we can talk about the specific scenario that we're seeing an issue?